students/hamin-Kim/user/views.py

```diff
@@ -30,5 +30,22 @@ def post(self, request):
             )
             return JsonResponse({"MESSAGE" : "SUCCESS"}, status = 201)
 
+        except KeyError:
+            return JsonResponse({"MESSAGE" : "KEY_ERROR"}, status = 400)
+
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if Users.objects.filter(email = data["email"]).exists() == False:
+                return JsonResponse({"MESSAGE" : "INVALID_USER"}, status = 401)
```

Comment:
```suggestion
            if not Users.objects.filter(email = data["email"]).exists():
                return JsonResponse({"MESSAGE" : "INVALID_USER"}, status = 401)
```
.exists()의 결과 값이 boolean이기 때문에 위와 같이 짜는 것이 더 파이썬스럽게 짰다고 볼 수 있습니다.